doc/howto/images_create.md

```diff
@@ -95,3 +95,11 @@ Usually, this includes the following data:
 For building your own images, you can use [LXD image builder](https://github.com/canonical/lxd-imagebuilder).
 
 See the [LXD image builder documentation](https://canonical-lxd-imagebuilder.readthedocs-hosted.com/en/latest/) for instructions for installing and using the tool.
+
+(images-repack-windows)=
+### Repack a Windows image
+
+You can run Windows VMs in LXD.
+To do so, you must repack the Windows ISO with [LXD image builder](https://github.com/canonical/lxd-imagebuilder).
```

Comment:
Can we link to https://canonical-lxd-imagebuilder.readthedocs-hosted.com/en/latest/ instead ?

Reply:
Actually, let's just get rid of this link, there's way too many in this section already. ;)